pytorch_lightning/plugins/precision/ipu_precision.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any
+
+from torch import Tensor
+
+from pytorch_lightning.plugins.precision.precision_plugin import PrecisionPlugin
+
+
+class IPUPrecisionPlugin(PrecisionPlugin):
```

Comment:
@awaelchli said before the next boilerplate precision plugin we should refactor to have the `PrecisionPlugin` inside of the `TrainingTypePlugin`, but maybe we can allow one more if this doesn't happen in time :P